cli/lib/kontena/callbacks/master/deploy/56_set_server_provider_after_deploy.rb

```diff
@@ -0,0 +1,27 @@
+module Kontena
+  module Callbacks
+    class SetServerProviderAfterDeploy < Kontena::Callback
+
+      include Kontena::Cli::Common
+
+      matches_commands 'master create'
+
+      def after
+        return unless command.exit_code == 0
+        return unless config.current_master
+        return unless config.current_master.name == command.result[:name]
+        return unless command.result[:provider]
+
+        require 'shellwords'
+
+        cmd = ['master', 'config', 'set', "server.provider=#{command.result[:provider]}"]
+        spinner "Setting Master configuration server.provider to '#{command.result[:provider]}'" do
+          Retriable.retriable do
+            Kontena.run(cmd.shelljoin)
```

Comment:
I'm starting to consider this `Kontena.run` harmful.. Why don't we just call method from the command object?